src/AELFFaucet.Web/appsettings.json

```diff
@@ -12,7 +12,11 @@
     "BaseUrlForSidechain": "https://tdvw-test-node.aelf.io",
     "PrivateKey": "b192e307da4c4e1a00eeed442a4b5e8e0b7b2c6f838d0472627b846e5280c51c",
     "SendCount": 100,
-    "NftSeedPrivateKey": "97a1747af1539a6b839c92227c8f5ca90ef0e7de70e5c2f46c64f27b12fabdd7"
+    "NftSeedPrivateKey": "97a1747af1539a6b839c92227c8f5ca90ef0e7de70e5c2f46c64f27b12fabdd7",
```

Comment:
All these keys should be removed and instead, for local development, use secrets.json instead.